schemas/isd.json

```diff
@@ -0,0 +1,236 @@
+                    "description": "Each coefficient used for the distortion equation in order of
+                                    increasing degrees." 
+                   }
+                },
+                "key" : {
+                   "type": "string",
+                   "description": "Key into LUT to determine equation to use."
+               }
+            },
+            "required": ["coefficients", "equation_key"]
+       },
+       "starting_detector_sample": {
+          "description": "The sample on the detector where the data starts",
+          "type": "number"
+       },
+       "starting_detector_line": {
+          "description": "The line on the detector where the data starts",
+          "type": "number"
+       },
+       "focal2pixel_samples": {
+          "description": "Transformation x coefficients from focal plane (mm) to detector pixel coordinates",
+          "type": "array",
+          "items": {
+             "type": "number"
+           }
+       },
+       "focal2pixel_lines": {
+          "description": "Transformation y coefficients from focal plane (mm) to detector pixel coordinates",
+          "type": "array",
+          "items": {
+             "type": "number",
+             "minItems": 3,
+             "maxItems": 3
+           }
+       },
+       "sensor_location": {
+          "description": "(X, Y, Z) location of sensor in body-fixed frame.",
+          "type": "array",
+          "items": {
+             "type": "array",
+             "items" : {
+                "type": "number",
+                "minItems": 3,
+                "maxItems": 3
+            }
+         }
+      },
+      "sensor_velocity" : {
+         "description": "(v_x, v_y, v_z) velocity of sensor in body-fixed frame.",
+         "type": "array",
+         "items": {
+            "type": "array",
+            "items" : {
+               "type": "number",
+               "minItems": 3,
+               "maxItems": 3
+            }
+         }
+      },
+      "sun_position" : {
+         "description": "(x, y,z) of the sun in body-fixed frame.",
+         "type": "array",
+         "items": {
+            "type": "array",
+            "items" : {
+               "type": "number",
+               "minItems": 3,
+               "maxItems": 3
+            }
+         }
+      },
+      "sun_velocity" : {
+         "description": "(v_x, v_y, v_z) velocity of sun in body-fixed frame.",
+         "type": "array",
+         "items": {
+            "type": "array",
+            "items" : {
+               "type": "number",
+               "minItems": 3,
+               "maxItems": 3
+            }
+         }
+      },
+      "sensor_orientation" : {
+         "description": "Orientation of the sensor provided in quaternions",
+         "type": "array",
+         "items": {
+            "type": "array",
+            "description": "Each quaternion is of the form: (scalar, vector, vector, vector)" 
```

Comment:
I'm not sure what is meant by vector in the context of a quaternion? Each quaternion is a vector of 4 scalars.  The important distinction is what the 4 values represent.  It may be better to link to an outside source that defines this for us because it takes some explanation. I'm going to try and find a good concise source.

Reply:
The naif whitepaper you linked to earlier I think explained both quaternions and all of the different ways quaternion formats may be different from each other due to the lack of a standard specification pretty well.

Reply:
@jessemapel Thanks for pointing that out: it should be 'vector component', rather than 'vector'. IIRC, that whitepaper referred to the components of a quaternion as (scalar, vector component, vector component, vector component).

Reply:
I did go ahead and link to that NAIF whitepaper. Let me know if you ended up finding anything different / better to describe quaternion formats?